css/css-syntax/tokenizing-and-parsing/input-byte-stream/i18n-css-syntax-bytes-001.html

```diff
@@ -20,13 +20,13 @@
 
 
 
-<div class="test"><div id="box" class="���">&#xA0;</div></div>
+<div class="test"><div id="box" class="???">&#xA0;</div></div>
```

Comment:
@xfq  Here and in other places, it looks like the the characters have been corrupted.